app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -68,25 +69,29 @@ class CustomChatInput extends ConsumerWidget {
 
     return FrostEffect(
       child: Container(
-        padding: const EdgeInsets.symmetric(vertical: 15, horizontal: 10),
+        padding: const EdgeInsets.symmetric(vertical: 15),
         decoration: BoxDecoration(color: Theme.of(context).colorScheme.surface),
-        child: Row(
-          children: [
-            const SizedBox(width: 1),
-            Icon(
-              Atlas.block_prohibited_thin,
-              size: 14,
-              color: Theme.of(context).unselectedWidgetColor,
-            ),
-            const SizedBox(width: 4),
-            Text(
-              key: noAccessKey,
-              L10n.of(context).chatMissingPermissionsToSend,
-              style: TextStyle(
-                color: Theme.of(context).unselectedWidgetColor,
+        child: Padding(
```

Comment:
Also it seemed necessary to move padding inner tree here to avoid overflow...

Reply:
I've seen cases where visually the effects look different comparably. Don't know how it's so but it is what it is...